src/proportional-bar/ProportionalBar.js

```diff
@@ -0,0 +1,132 @@
+import { sum } from "d3-array";
+import { format } from "d3-format";
+import PropTypes from "prop-types";
+import React, { useState } from "react";
+import Measure from "react-measure";
+import ResponsiveOrdinalFrame from "semiotic/lib/ResponsiveOrdinalFrame";
+import styled from "styled-components";
+import ColorLegend from "../color-legend";
+import Tooltip from "../tooltip";
+
+const ProportionalBarContainer = styled.figure`
+  height: 100%;
+  margin: 0;
+  position: relative;
+  width: 100%;
+`;
+
+const ProportionalBarChartWrapper = styled.div`
+  height: 100%;
+  position: relative;
+  z-index: ${(props) => props.theme.zIndex.base + 1};
+`;
+
+const ProportionalBarMetadata = styled.figcaption`
+  align-items: baseline;
+  bottom: 0;
+  display: flex;
+  justify-content: space-between;
+  padding-top: 10px;
+  position: absolute;
+  width: 100%;
+  z-index: ${(props) => props.theme.zIndex.base};
+`;
+const ProportionalBarTitle = styled.div`
+  color: ${(props) => props.theme.colors.body};
+  flex: 0 0 auto;
+  font: ${(props) => props.theme.fonts.body};
+  margin-right: 15px;
+`;
+
+const ProportionalBarTooltipText = styled.p`
+  ${(props) => (props.bold ? `font: ${props.theme.fonts.bodyBold};` : "")}
+  line-height: 2;
+  margin: 0;
+  white-space: nowrap;
+`;
+
+// if the legend doesn't wrap it should be roughly this size;
+// the chart height will adjust as necessary
+const INITIAL_METADATA_HEIGHT = 28;
+
+const formatAsPct = format(".0%");
+
+export default function ProportionalBar({ data, title }) {
```

Comment:
I didn't notice this component being used anywhere in this PR, but I'm assuming that's just because it'll get pulled in for https://github.com/Recidiviz/public-dashboard/issues/11?